javascript/ql/lib/semmle/javascript/security/performance/ReDoSUtil.qll

```diff
@@ -508,7 +550,7 @@ private State before(RegExpTerm t) { result = Match(t, 0) }
 /**
  * Gets a state the NFA may be in after matching `t`.
  */
-private State after(RegExpTerm t) {
+State after(RegExpTerm t) {
```

Comment:
Why make this public? (leftover from testing?)

Reply:
I need it to be public in the query I'm developing. 
I just copy-pasted the `ReDoSUtil.qll` file from my development of that query into this separate PR, so that's why I removed the `private` modifier.